interfaces/acados_template/acados_template/acados_ocp_solver.py

```diff
@@ -985,9 +985,9 @@ def __init__(self, acados_ocp, json_file='acados_ocp_nlp.json', simulink_opts=No
 
 
     def __get_pointers_solver(self):
-        """
-        Private function to get the pointers for solver
-        """
+        # """
+        # Private function to get the pointers for solver
+        # """
```

Comment:
Why do you comment the docstring?

Reply:
It was generated on the documentation page very prominently, which I didn't like.

Reply:
Ok, I understand. We might consider to keep only line 989 as a standard comment with #, and delete the double quotes.